.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+### Thank you for your Pull Request! 
+
+We have developed a Pull Request template to aid you and our reviewers.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the xportr codebase remains robust and consistent.  
+
```

Comment:
Should there be an empty section that developer can use to describe the changes?

```suggestion

### Changes description

_(descriptions of changes)_

```

Reply:
Yes this is very good idea.  We should make it optional as sometimes the changes are self-explanatory.